app/src/test/java/com/example/marvelapp/usecase/GetCharacterByIdUseCaseTest.kt

```diff
@@ -20,17 +18,17 @@ import org.mockito.kotlin.whenever
 @RunWith(MockitoJUnitRunner::class)
 class GetCharacterByIdUseCaseTest {
 
-    private lateinit var getCharacterByIdUseCase: GetCharacterByIdUseCase
+    private lateinit var getCharacterByIdUseCase: com.example.domain.usecase.GetCharacterByIdUseCase
     private val characterDetailService: CharacterService = mock()
-    private val database: MarvelRepository = mock()
+    private val database: com.example.data.database.MarvelRepository = mock()
```

Comment:
import the MarvelRepository instead of put here all the path

Reply:
done